server/service/src/sync/sync_status/test.rs

```diff
@@ -389,25 +389,49 @@ fn get_push_and_error_sync_status_tester(service_provider: Arc<ServiceProvider>)
                         assert!(push_status.finished.is_none());
 
                         assert_eq!(push_status.done, Some(0));
-                        assert_eq!(push_status.total, Some(3));
+                        assert_eq!(push_status.total, Some(7));
 
                         false
                     }
                     1 => {
                         assert_eq!(push_status.done, Some(1));
-                        assert_eq!(push_status.total, Some(3));
+                        assert_eq!(push_status.total, Some(7));
 
                         false
                     }
                     2 => {
                         assert_eq!(push_status.done, Some(2));
-                        assert_eq!(push_status.total, Some(3));
+                        assert_eq!(push_status.total, Some(7));
 
                         false
                     }
                     3 => {
                         assert_eq!(push_status.done, Some(3));
-                        assert_eq!(push_status.total, Some(3));
+                        assert_eq!(push_status.total, Some(7));
+
+                        false
+                    }
+                    4 => {
+                        assert_eq!(push_status.done, Some(4));
+                        assert_eq!(push_status.total, Some(7));
+
+                        false
+                    }
+                    5 => {
+                        assert_eq!(push_status.done, Some(5));
+                        assert_eq!(push_status.total, Some(7));
+
+                        false
+                    }
+                    6 => {
+                        assert_eq!(push_status.done, Some(6));
+                        assert_eq!(push_status.total, Some(7));
+
+                        false
+                    }
+                    7 => {
+                        assert_eq!(push_status.done, Some(7));
+                        assert_eq!(push_status.total, Some(7));
```

Comment:
I am not familiar with this test. Where are these extra entries coming from? Was there a change in the mock data? what is an iteration here?

Reply:
Coming from adding name to changelog

Reply:
I think here and other change_log tests, it's already somewhat difficult to see what is being tested (I think this test is quite complex, I was never happy with it). Sine it's already complex I would suggest just cleaning the log here:
https://github.com/openmsupply/open-msupply/blob/3ee43e06d8d277588c48a5310ea3fc23ebd54df4/server/service/src/sync/sync_status/test.rs#L85-L87

And after setting up base data in changelog/test.md, i.e. here:

https://github.com/openmsupply/open-msupply/blob/3ee43e06d8d277588c48a5310ea3fc23ebd54df4/server/repository/src/db_diesel/changelog/test.rs#L133-L134

Reply:
Would mean you'll need to add a drop all changelog methods (with test flag) but I think it would be simpler and less code overall

Reply:
Can this be moved to another issue?

Reply:
@andreievg thanks for the clarification! seems reasonable to me

@roxy-dao it was just a question from my side, so no need to do it here